settings.py

```diff
@@ -2345,8 +2347,10 @@ def _determine_couch_databases(couch_databases):
 
 PACKAGE_MONITOR_REQUIREMENTS_FILE = os.path.join(FILEPATH, 'requirements', 'requirements.txt')
 
-IS_LOCATION_CTE_ENABLED = UNIT_TESTING or SERVER_ENVIRONMENT in [
-    'localdev',
-    'changeme',  # default value in localsettings.example.py
-    'staging',
-]
+if IS_LOCATION_CTE_ENABLED is None:
+    IS_LOCATION_CTE_ENABLED = UNIT_TESTING or SERVER_ENVIRONMENT in [
+        'localdev',
+        'changeme',  # default value in localsettings.example.py
+        'staging',
+        'softlayer',
+    ]
```

Comment:
nit: This is a little strange - is it controlled by ansible/localsettings or is it hardcoded in HQ?  Shouldn't something like softlayer have it turned off/on by the former method so it can be reverted without a deploy?

Reply:
As of this PR, it's possible to override this by setting `IS_LOCATION_CTE_ENABLED` in localsettings.py, so it can be controlled by ansible if need be, but is not by default. I did it this way because I wanted to have it enabled/disabled on some envs when the PR is merged, and avoid extra steps of doing extra ansible deploys to roll this out in most cases. I'm just using this while monitoring query times with datadog, but expect it to go away when I'm no longer doing that.